### PR TITLE
JsonHandler is responsible for processing json sent from client.

### PR DESCRIPTION
**Pull request by maciejkocur**
For now it can response for start Game. It also hnadles not recognize json.

### CODE REVIEW

File: server/src/main/java/io/github/expansionteam/battleships/JsonHandler.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import org.json.JSONObject;
+
+public class JsonHandler {
+
+    String resolveAction(String str) {
+        JSONObject jsonObj = new JSONObject(str);
+        String type = jsonObj.getString("type");
+        JSONObject jsonResponse = new JSONObject();
+
+        switch (type) {
```
Comment (danone771909) [inline server/src/main/java/io/github/expansionteam/battleships/JsonHandler.java]: why switch loop, if we have only one case ?

File: server/src/main/java/io/github/expansionteam/battleships/PlayerThread.java [hunk 2]
```diff
@@ -25,17 +29,19 @@ void setThreadToInform(PlayerThread playerThread) {
...
 
             while (true) {
 
-                str = dis.readUTF();
-                System.out.println(str);
-                dos.writeUTF("response: " + str);
-                dos.flush();
+                jsonRequest = dataInputStream.readUTF();
+                jsonResponse = jsonHandler.resolveAction(jsonRequest);
+                System.out.println(jsonResponse);
```
Comment (danone771909) [inline server/src/main/java/io/github/expansionteam/battleships/PlayerThread.java]: why sout? you should use Logger.

File: server/src/test/java/io/github/expansionteam/battleships/JsonHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        assertEquals(actualResponse, expectedResponseJson);
+    }
+
+    @Test
+    public void responsesToNotRecognizeEvent() {
+
+        // given
+        JsonHandler jsonHandler = new JsonHandler();
+        String requestJson = "{\"type\":\"NoSuchEvent\"}";
+        System.out.println(requestJson);
```
Comment (danone771909) [inline server/src/test/java/io/github/expansionteam/battleships/JsonHandlerTest.java]: sout?

File: server/src/main/java/io/github/expansionteam/battleships/JsonHandler.java (hunk 1 above)
Comment (maciejkocur) [inline server/src/main/java/io/github/expansionteam/battleships/JsonHandler.java]: You're right. I prepared it to next cases that not existing yet.

Comment (maciejkocur) [inline server/src/main/java/io/github/expansionteam/battleships/JsonHandler.java]: You're right. I prepared it to next cases that not existing yet. Now there is default case.

File: server/src/main/java/io/github/expansionteam/battleships/PlayerThread.java (hunk 2 above)
Comment (maciejkocur) [inline server/src/main/java/io/github/expansionteam/battleships/PlayerThread.java]: deleted

File: server/src/test/java/io/github/expansionteam/battleships/JsonHandlerTest.java (hunk 3 above)
Comment (maciejkocur) [inline server/src/test/java/io/github/expansionteam/battleships/JsonHandlerTest.java]: deleted

**Comment by clinton56** [comment]
I accept

**Pull request merged**